src/Workflow/MoveJatsAssets.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentStore\Workflow;
```

Comment:
This will need to be in a separate library, but not sure where yet.

Reply:
Could make it not JATS specific, but would probably have to emit an event to allow something to do https://github.com/libero/content-store/blob/0093dce66220d7a3a3f6017d5382838754eaa534/src/Workflow/MoveJatsAssets.php#L129-L132